keras_cv/models/backbones/csp_darknet/csp_darknet_backbone.py

```diff
@@ -0,0 +1,398 @@
+        )
+
+        base_channels = stackwise_channels[0] // 2
+
+        inputs = utils.parse_model_inputs(input_shape, input_tensor)
+
+        x = inputs
+        if include_rescaling:
+            x = layers.Rescaling(1 / 255.0)(x)
+
+        # stem
+        x = Focus(name="stem_focus")(x)
+        x = DarknetConvBlock(
+            base_channels, kernel_size=3, strides=1, name="stem_conv"
+        )(x)
+
+        pyramid_level_inputs = {}
+        for index, (channel, depth) in enumerate(
+            zip(stackwise_channels, stackwise_depth)
+        ):
+            x = ConvBlock(
+                channel,
+                kernel_size=3,
+                strides=2,
+                name=f"dark{index + 2}_conv",
+            )(x)
+
+            if index == len(stackwise_depth) - 1:
+                x = SpatialPyramidPoolingBottleneck(
+                    channel,
+                    hidden_filters=channel // 2,
+                    name=f"dark{index + 2}_spp",
+                )(x)
+
+            x = CrossStagePartial(
+                channel,
+                num_bottlenecks=depth,
+                use_depthwise=use_depthwise,
+                residual=(index != len(stackwise_depth) - 1),
+                name=f"dark{index + 2}_csp",
+            )(x)
+            pyramid_level_inputs[index + 2] = x.node.layer.name
+
+        super().__init__(inputs=inputs, outputs=x, **kwargs)
+        self.pyramid_level_inputs = pyramid_level_inputs
+
+        self.stackwise_channels = stackwise_channels
+        self.stackwise_depth = stackwise_depth
+        self.include_rescaling = include_rescaling
+        self.use_depthwise = use_depthwise
+        self.input_tensor = input_tensor
+
+    def get_config(self):
+        config = super().get_config()
+        config.update(
+            {
+                "stackwise_channels": self.stackwise_channels,
+                "stackwise_depth": self.stackwise_depth,
+                "include_rescaling": self.include_rescaling,
+                "use_depthwise": self.use_depthwise,
+                "input_shape": self.input_shape[1:],
+                "input_tensor": self.input_tensor,
+            }
+        )
+        return config
+
+    @classproperty
+    def presets(cls):
+        """Dictionary of preset names and configurations."""
+        return copy.deepcopy(backbone_presets)
+
+    @classproperty
+    def presets_with_weights(cls):
+        """Dictionary of preset names and configurations that include
+        weights."""
+        return copy.deepcopy(backbone_presets_with_weights)
+
+
+ALIAS_DOCSTRING = """CSPDarkNetBackbone model with {stackwise_channels} channels
+    and {stackwise_depth} depths.
+
+    Reference:
+        - [YoloV4 Paper](https://arxiv.org/abs/1804.02767)
+        - [CSPNet Paper](https://arxiv.org/pdf/1911.11929)
+        - [YoloX Paper](https://arxiv.org/abs/2107.08430)
+        - [YoloX implementation](https://github.com/ultralytics/yolov3)
```

Comment:
Looks like the erroneous YOLOX link is still in this version of the docstring

Reply:
Good catch! I've only removed the base class one.